charts/prometheus-kafka-exporter/values.yaml

```diff
@@ -103,6 +105,23 @@ tls:
   # mountPath: /secret/certs
   # secretName: <name of an existing secret>
 
+sasl:
+  enabled: false
+  handshake: true
+  scram:
+    enabled: false
```

Comment:
Can you add a way to declare an existing secret for scram ?

Reply:
done. can you think of a more concise way?

Reply:
it's not shorter but you can use a template in the _helpers.yaml to write the if statement only once